packages/assistify-ai/models/AssistifySmarti.js

```diff
@@ -0,0 +1,34 @@
+class AssistifySmarti extends RocketChat.models._Base {
```

Comment:
@ThomasRoehl we're not using the rocketchat-external-messages-model anymore, but an own one.